.jenkins/sarus_run_test.sh

```diff
@@ -1,9 +1,40 @@
+## sarus_run_test.sh
+##
+## Bash script that runs a C12 reference test using two Docker images (one with
+## Serialization support and one without).  
+##
+## Sarus cannot pull Docker images directly from the Google Container Registry.
+## The work-around is to download compressed archive files of the desired
+## Docker images from Google Storage, uncompress the archives and add them in
+## a local Docker repo on Daint.  The Docker images can then be run as normal
+## via Sarus.
+##
+## User is required to enter a valid tag as an input argument.  This tag will
+## be appended to the two Docker images used for the C12 reference tests. The 
+## Docker image with Serialization support enabled will have '-serialize' 
+## appended to its image name.
+##
+##   Mark Cheeseman, VCM 
+##   October 8, 2020
+##
+##
+## C12 runs and MD5 checksum verification is performed under pytest.  The
+## user needs to tell pytest that Sarus will be used for Docker image 
+## deployment via the --image_runner option.
+##
+##    Jeremy McGibbon, VCM
+##    October 8, 2020
+
 #!/bin/bash
```

Comment:
I think this has to be the first line, no?

Reply:
moved